contrib/cron-jobs/qualle.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# How often should this run?
+#  Once a day max
+
+# crontab entry for this (proposal):
+# # m h  dom mon dow   command
+# 02  01 *   *   *     ( cd /tmp; /my/path/to/this/script )
+#
+
+cd /tmp
+mkdir qualle
+cd qualle
+
+echo "Download IPv4 table from 'qualle.cert.at'."
+whois -h qualle.cert.at block origin v4table > ipasn.dat
```

Comment:
actually that's not really public :) But ok. No big secret either. Anyway... maybe let's abstract this by using a variable. Like:

```bash
BGPROUTER=....

...
whois -h $BGPROUTER block origin v4table > ipasn.dat
```